tests/unit/cdomains/intDomainTest.ml

```diff
@@ -218,9 +219,34 @@ struct
     assert_equal imax (I.widen ik i65536 i65537);
     assert_equal imax (I.widen ik i65536 imax)
 
+  let test_interval_narrow _ =
+    GobConfig.set_bool "ana.int.interval_threshold_widening" true;
+    GobConfig.set_string "ana.int.interval_threshold_widening_constants" "comparisons";
```

Comment:
Is this list ever populated here?

Reply:
It is and does work for the widening test above. The list includes some defaults (powers of 2) in addition to whatever may be collected from the program.